README.md

```diff
@@ -5,140 +5,27 @@
-            forced_allow_origin_value: ~
-            skip_same_as_origin: true
-        paths:
-            '^/api/':
-                allow_origin: ['*']
-                allow_headers: ['X-Custom-Auth']
-                allow_methods: ['POST', 'PUT', 'GET', 'DELETE']
-                max_age: 3600
-            '^/':
-                origin_regex: true
-                allow_origin: ['^http://localhost:[0-9]+']
-                allow_headers: ['X-Custom-Auth']
-                allow_methods: ['POST', 'PUT', 'GET', 'DELETE']
-                max_age: 3600
-                hosts: ['^api\.']
-```
-
-`allow_origin` and `allow_headers` can be set to `*` to accept any value, the
-allowed methods however have to be explicitly listed. `paths` must contain at least one item.
-
-`expose_headers` can be set to `*` to accept any value as long as `allow_credentials` is `false`
-[as per the specification](https://developer.mozilla.org/en-US/docs/Web/HTTP/Headers/Access-Control-Expose-Headers).
-
-If `origin_regex` is set, `allow_origin` must be a list of regular expressions matching
-allowed origins. Remember to use `^` and `$` to clearly define the boundaries of the regex.
-
-By default, the `Access-Control-Allow-Origin` response header value is 
-the `Origin` request header value (if it matches the rules you've defined with `allow_origin`),
-so it should be fine for most of use cases. If it's not, you can override this behavior 
-by setting the exact value you want using `forced_allow_origin_value`.
-
-Be aware that even if you set `forced_allow_origin_value` to `*`, if you also set `allow_origin` to `http://example.com`,
-only this specific domain will be allowed to access your resources.
-
-> **Note:** If you allow POST methods and have 
-> [HTTP method overriding](http://symfony.com/doc/current/reference/configuration/framework.html#http-method-override)
-> enabled in the framework, it will enable the API users to perform PUT and DELETE 
-> requests as well.
-
-## Cookbook
-
-### How to ignore preflight requests on New Relic?
-
-On specific architectures with a mostly authenticated API, preflight request can represent a huge part of the traffic.
-
-In such cases, you may not need to monitor on New Relic this traffic which is by the way categorized automatically as
-`unknown` by New Relic.
-
-A request listener can be written to ignore preflight requests:
-```php
-use Symfony\Component\HttpKernel\Event\FilterResponseEvent;
-
-class PreflightIgnoreOnNewRelicListener
-{
-    public function onKernelResponse(FilterResponseEvent $event)
-    {
-        if (!extension_loaded('newrelic')) {
-            return;
-        }
-
-        if ('OPTIONS' === $event->getRequest()->getMethod()) {
-            newrelic_ignore_transaction();
-        }
-    }
-}
-```
-
-Register this listener, and voilà!
+The bundle should be automatically enabled by [Symfony Flex][1]. If you don't use
+Flex, you'll need to enable it manually as explained [in the docs][2].
 
 ## License
```

Comment:
I think it'd be good to get a section above here like:

```
## Usage

See [the documentation][2] for usage instructions.
```

Just so you at least have a way from the readme/packagist.org to find the docs, if you happen to look at the bundle directly and not browsing on symfony site.

Reply:
I totally forgot about this. Sorry. It's fixed now. Thanks.